packages/components/src/VirtualizedList/CellActions/CellActions.component.js

```diff
@@ -1,18 +1,20 @@
 import React, { PropTypes } from 'react';
-import classnames from 'classnames';
```

Comment:
Nothing, just the IDE that points me out that there is typo in the word `classnames` :)